seacrowd/sea_datasets/lio_and_central_flores/processing.py

```diff
@@ -0,0 +1,55 @@
+import re
+
+def parse_text(data):
+    cleaned_data = [re.sub(r'^\ufeff?\d+\t|\b\d+\)\s*', '', re.sub(r'\t+', '\t', text)).replace('\t\n', '').replace('\ufeff', '') for text in data]
```

Comment:
Not sure how if it's easy to maintain this regex. Perhaps you can put a docstring somewhere that explains what happens here?